src/main/java/io/nats/client/impl/NatsConnectionReader.java

```diff
@@ -352,7 +355,7 @@ public String opFor(char[] chars, int length) {
             } else if (chars[0] == '-' && 
                         (chars[1] == 'E' || chars[1] == 'e') &&
                         (chars[2] == 'R' || chars[2] == 'r') && 
-                        (chars[3] == 'R' || chars[3] == 'R')) {
+                        (chars[3] == 'R' || chars[3] == 'r')) {

```

Comment:
also noticed this probably needs to be lowercase :)

Reply:
very nice catch on the R->r, my brain never even thought to see that.

I started looking at the PR and realized i am handling a bunch of these states in handleCommunicationIssue already, so will fix it there:
```
            if (this.connecting || this.disconnecting || this.status == Status.CLOSED || this.isDraining()) {
```

As a result I will likely close this PR without a merge, but the issue will be fixed in 2.4.2, which I am working on as I type. When I get the change in to connection.handleCommunicationIssue, i will ping to have you try it out before I release 2.4.2

Reply:
ok, just let me know when 2.4.2 is ready, thanks!